SPEC.md

```diff
@@ -2488,7 +2502,7 @@ Example 2:
   }
 
   output {
-    File outputFile = ${output_file_name}
+    File outputFile = output_file_name
```

Comment:
AFAIK, this has always been the correct syntax for outputs:

 - `output_file_name` is a variable, and you're assigning its value to `outputFile`.
 - `${output_file_name}` is a string interpolation that you can use inside a command block. I don't know that it would work outside a command block and would argue that it shouldn't.
 - You might be able to do something like `= "${output_file_name}"` but this is a needless indirection. You'd be saying "insert this String into another String, and then use that". It works, but you might as well use the original String in the first place